scripts/viewer2.py

```diff
@@ -0,0 +1,880 @@
+import sys
+import PyQt4.QtGui as QtGui
+import PyQt4.QtCore as QtCore
+
+import sys
+from msct_parser import Parser
+from msct_image import Image
+from bisect import bisect
+from numpy import arange, max, pad, linspace, mean, median, std, percentile
+import numpy as np
+from msct_types import *
+
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+from matplotlib.lines import Line2D
+from matplotlib import cm
+import sct_utils as sct
+from time import time
+from copy import copy
+from msct_image import Image
+
+from matplotlib.widgets import Slider, Button, RadioButtons
+from matplotlib.backends.backend_qt4agg import FigureCanvasQTAgg as FigureCanvas
+
+from abc import ABCMeta, abstractmethod
+
+import webbrowser
+
+
+class Observer(object):
```

Comment:
You mentioned that Observer and Observable are not used. Is this still the case?